Source/Classes/AnimatedImages/PINCachedAnimatedImage.m

```diff
@@ -302,7 +305,7 @@ - (void)l_cacheFrame:(NSUInteger)frameIndex
     if ([_cachedOrCachingFrames containsIndex:frameIndex] == NO && _cacheCleared == NO) {
         PINLog(@"Requesting: %lu", (unsigned long)frameIndex);
         [_cachedOrCachingFrames addIndex:frameIndex];
-        _playbackReady++;
+        _frameRenderCount++;
```

Comment:
Maybe we could set _playbackReady here "proactively", instead of leaving it NO until access to the getter method causes it to check _frameRenderCount?

Reply:
Agree with @appleguy to we could set  the `_playbackReady` in here, that said I don't think we will win anything in here, in fact I would said having fewer state changes in the code base is better e.g. if we can derive state from other variables like in this place the `playbackReady` state will result in fewer bugs and maintenance costs later.